Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM python:3
 
-RUN pip install apprise j2cli
+RUN set -x \
+    \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends jq \
```

Comment:
[hadolint](https://github.com/hadolint/hadolint) complained about not pinning neither `jq` nor the `pip` installed dependencies. Still, I ignored it since `jq` has not change since 2018. wdyt?